Sources/Core/AWSSDKEventStreamsAuth/AWSSigV4Signer+EventStreams.swift

```diff
@@ -0,0 +1,55 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Smithy
+import SmithyHTTPAuthAPI
+import SmithyEventStreamsAPI
+import SmithyEventStreamsAuthAPI
+import AWSSDKHTTPAuth
+import struct Foundation.Data
+import AwsCommonRuntimeKit
+
+extension AWSSigV4Signer: MessageDataSigner {
```

Comment:
This extension was previously defined right on the signer.  Moved it out separate so it could be packaged with event streams auth.